server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -183,7 +187,10 @@ public Invocations getInvocations() {
          return CompletableFuture.completedFuture(responseBuilder.build());
       }
       return CompletableFuture.supplyAsync(() -> {
-         restCacheManager.getCacheManagerAdmin(request).removeCache(cacheName);
+         Subject.doAs(request.getSubject(), (PrivilegedAction<Void>) () -> {
```

Comment:
RestCacheManager.getCacheManagerAdmin() should already return a privileged RemoteCacheAdmin so remove all these `Subject.doAs` invocations

Reply:
ok! yes